docs/examples/general/data_loading/numpy_reader.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        data = fn.readers.numpy(device='cpu', file_root=file_root, files=files_arg,
</pre><p><code># pass the list of files directly as files keyword argument</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>